code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -416,6 +417,7 @@
 		load = null
 		M.layer = initial(M.layer)
 		M.pixel_y = initial(M.pixel_y)
+		buckle_prevents_pull = FALSE
```

Comment:
Why toggle this ? Shouldn't this be always true ?

Reply:
To be honest I wasn't sure, I guessed since it was previously false it should return to being false in case there were some unintentional side effects, but makes sense to keep it at true I suppose.

Reply:
Just make it the default for mulebots, i can't think of any unwanted side-effects from this.